zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -91,6 +91,11 @@ public Note getNote() {
     return note;
   }
 
+  public boolean isEnabled() {
+    Boolean enabled = (Boolean) config.get("enabled");
+    return enabled == null || enabled.booleanValue();
```

Comment:
The default behavior (when the value "enabled" is undefined) need to be 'enabled'.
Some notebook that created before this "enabled" option introduced to Zeppelin, will have null value for it. And it think it should be considered as "enabled" in this case.

Make sense?